app-devel/judy/autobuild/patches/05_remove_automake_version_restriction.patch

```diff
@@ -0,0 +1,11 @@
+--- judy-1.0.5/bootstrap	2007-05-14 21:10:51.000000000 +0000
++++ judy-1.0.5-new/bootstrap	2024-08-11 21:13:54.643575106 +0000
+@@ -8,7 +8,7 @@
+ autoheader
+ #add --include-deps if you want to bootstrap with any other compiler than gcc
+ #automake --add-missing --copy --include-deps
+-automake-1.9 --add-missing --force --copy
++automake --add-missing --force --copy
+ #autoconf2.50
+ autoconf
+ rm -f config.cache
```

Comment:
Please consider tracking these at AOSC-Tracking. I have sent you an invite to that organisation since you've missed it...